modules/document_repository/jsx/editFormIndex.js

```diff
@@ -0,0 +1,21 @@
+/* global ReactDOM */
+
+import DocEditForm from './editForm';
+const args = QueryString.get(document.currentScript.src);
+
+$(function() {
```

Comment:
similarly this should be an addEventListener instead of jQuery.

Reply:
done